docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: "3"
+services:
+
+  mapping-service:
+    build:
+      context: .
+      dockerfile: tests/resources/Dockerfile
+    ports:
+      - 8888:8888
+    volumes:
+      - ./src:/app/src
+      - ./tests:/app/tests
+
+  blazegraph:
+    image: metaphacts/blazegraph-basic:2.2.0-20160908.003514-6-jetty9.4.44-jre8-45dbfff
+    ports:
+      - 8889:8080
```

Comment:
is there a reason we can't just use the default ports for each service as we expose outside of docker?